### PR TITLE
Openshift Istio via Service Mesh Operator:

### PR DESCRIPTION
**Pull request by gcs278**
- Added ISTIO_OSSM Env for installing Openshift Istio Service Mesh Operator and utilizing it for the demo
- Limited to only using the default Envoy Deployment (istio-ingressgateway)
- Limited to putting the Control Plane in the same namespace as our Gateway object (gwapi)

### CODE REVIEW

**Comment by gcs278** [comment]
@frobware want to review this?

File: setup-itsio-example.sh [hunk 1]
```diff
@@ -2,10 +2,70 @@
 
 set -eu
 
+# ENV Defaults
+: "${ISTIO_API_DEMO:=false}"
```
Comment (frobware) [inline setup-itsio-example.sh:7]: An alternative way for exporting all of these would be:

```sh
set -a
: "${ISTIO_API_DEMO:=false}"
: "${ISTIO_HOST_NETWORKING:=false}"
...
set +a
```

**Review by frobware — COMMENTED** [review]
LGTM, but I didn't try to run the instructions live on a cluster this time. Regarding some of the scripts, I rely on the ever excellent https://www.shellcheck.net/ for consistency and correctness.

File: setup-itsio-example.sh (hunk 1 above)
Comment (gcs278) [reply setup-itsio-example.sh:7]: brilliant thanks!

**Pull request merged**